content/general/concepts/file-paths/file-paths.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: 'File Paths'
+Description: 'A file path describes the location of a file in a web sites folder structure. They are used to link to external files.'
```

Comment:
```suggestion
Description: 'A file path describes the location of a file in a web site's folder structure. They are used to link to external files.'
```

Reply:
Also: the general subject of file paths are important to the file structure on any computer, not just web applications.
So something like this would be better: "A file path describes the location of a file in a computer's folder structure."